cmd/format-erasure.go

```diff
@@ -386,7 +373,10 @@ func saveFormatErasure(disk StorageAPI, format *formatErasureV3, heal bool) erro
 
 	disk.SetDiskID(diskID)
 	if heal {
-		return saveHealingTracker(disk, diskID)
+		ctx := context.Background()
+		ht, err := newHealingTracker(disk)
+		logger.LogIf(ctx, err)
+		return ht.save(ctx)
```

Comment:
potential for having `ht` can be `nil` ? if GetDiskID() failed? - you can pass the diskID here to newHealingTracker() instead of calling disk.GetDiskID()

Reply:
Yeah, I'll simplify.